ukbb_qc/release/prepare_vcf_data_release.py

```diff
@@ -376,6 +379,15 @@ def main(args):
             ht = ht.annotate(freq=ht.freq[:22])
             ht = ht.annotate_globals(freq_meta=ht.freq_meta[:22])
 
+            # Add AS_SB_TABLE, AS_QUALapprox here because it is missing from the release HT
```

Comment:
Make a note for if/when we release the MT and/or HTs that we should add these. I'm assuming `AS_QUALapprox` was removed because it's used for `qual` right? `AS_SB_TABLE` wasn't previously in gnomAD either, we added it for the v3.1.1 release by request from a user. In gnomad_qc I also needed to do this: https://github.com/broadinstitute/gnomad_qc/blob/33177b82e703529928022aa8e19a2feb27c20830/gnomad_qc/v3/create_release/prepare_vcf_data_release.py#L747, just making sure that isn't needed? What does `AS_SB_TABLE` look like at this point?

Reply:
actually, it looks like we used the site level QUALapprox for qual:
```
## VCF release
We decided to include the `QUALapprox` score (labeled as `QUAL`) in the VCF release rather than the traditional `QUAL` score for consistency with the UKBB browser. Note that `QUALapprox` and `QUAL` scores are very similar for most variants, but the formula for `QUALapprox` (sum of `PL[0]` values for all samples) is easier to interpret than `QUAL`'s (which involves a complex expectation-maximization algorithm).
``` 
(from our release readme)
also https://github.com/broadinstitute/ukbb_qc/blob/master/ukbb_qc/release/prepare_internal_data_release.py#L103

I will need to look at AS_SB_TABLE, I'm guessing we will need to do a similar reformat here

Reply:
me too, it took me a while to track down, but it looks like AS_QUALapprox wasn't part of `AS_FIELDS` when we created the 455k: https://github.com/broadinstitute/gnomad_methods/commit/6b7d34dcddcd4696f21319fb3da6480618d30bd4#diff-b5a0be260e1a0153ac1535bd57a840df4cef8fadb1b0349fabbced3881d1f85fR49 (hopefully this link works)

Reply:
(so I guess I didn't roll back gnomad methods far enough to have the code compatible with VCF export even if it should be far enough for the validity checks)

Reply:
Well, I guess it's catching some other things though. It was before we completed all the v3.1 technical backlog because that is likely when `AS_QUALapprox` got added